templates/auth/login.html

```diff
@@ -13,36 +13,44 @@
     </div>
   {% endif %}
 
-  <form class="card" style="border-radius: 2%" method="post">
-    {{ form.csrf_token }}
+  <div class="card" style="border-radius: 2%">
     <div class="card-body p-6">
       <h1 class="card-title">Welcome back!</h1>
-      <div class="form-group">
-        <label class="form-label">Email address</label>
-        {{ form.email(class="form-control", type="email", autofocus="true") }}
-        {{ render_field_errors(form.email) }}
-      </div>
-
-      <div class="form-group">
-        <label class="form-label">
-          Password
-        </label>
-        {{ form.password(class="form-control", type="password") }}
-        {{ render_field_errors(form.password) }}
-        <div class="text-muted">
-          <a href="{{ url_for('auth.forgot_password') }}" class="small">
-            I forgot my password
-          </a>
+      <form method="post">
+        {{ form.csrf_token }}
+
+        <div class="form-group">
+          <label class="form-label">Email address</label>
+          {{ form.email(class="form-control", type="email", autofocus="true") }}
+          {{ render_field_errors(form.email) }}
         </div>
-      </div>
 
-      <div class="form-footer">
-        <button type="submit" class="btn btn-primary btn-block">Log in</button>
-      </div>
+        <div class="form-group">
+          <label class="form-label">
+            Password
+          </label>
+          {{ form.password(class="form-control", type="password") }}
+          {{ render_field_errors(form.password) }}
+          <div class="text-muted">
+            <a href="{{ url_for('auth.forgot_password') }}" class="small">
+              I forgot my password
+            </a>
+          </div>
+        </div>
 
+        <div class="form-footer">
+          <button type="submit" class="btn btn-primary btn-block">Log in</button>
+        </div>
 
+      </form>
+      {% if connect_with_proton %}
+        <div class="text-center my-2 text-gray"><span>or</span></div>
+        <a class="btn btn-primary btn-block mt-2" style="border-color:#6d4aff;background-color:white;color:#6d4aff" href="{{ url_for("auth.proton_login") }}">Log in with Proton</a>
+        <div class="text-center my-2 text-gray"><span>SimpleLogin and ProtonMail have joined forces to protect your inbox from spam and phising. <br /><a href="https://simplelogin.io/blog/simplelogin-join-proton/">Read more</a></span></div>
```

Comment:
As seen with Andy, we can remove this section "SimpleLogin and ProtonMail have joined forces ..." to not overcharge the login screen :)

Reply:
Text removed